utils/api.js

```diff
@@ -26,4 +26,22 @@ export const getCommentsByArticleId = (id)=>{
     }).catch((err)=>{
         console.log(err)
     })
+}
+
+export const increaseArticleVote = (id)=>{
+    const likeButton = {inc_votes: 1}
+    return app.patch(`/articles/${id}`, likeButton).then(({data}) =>{
+        return data.article
+    }).catch((err) =>{
+        console.log(err)
+    })
+}
+
+export const decreaseArticleVote =(id) =>{
+const dislikeButton = {inc_votes:-1}
+return app.patch(`/articles/${id}`, dislikeButton).then(({data}) =>{
+    return data.article
+}).catch((err) =>{
+    console.log(err)
+})
```

Comment:
Nice separation into a separate API function .You could refactor this into one function which takes the amount it should be incremented by as an argument, along with the article id